source/crud.txt

```diff
@@ -0,0 +1,48 @@
+.. _php-crud-operations:
+
+===============
+CRUD Operations
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :description: Learn how to use the {+library-short+} to read and write MongoDB data.
+   :keywords: usage examples, query, find, code example, save, create
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+   
+   Insert Documents </crud/insert>
+   Query Documents </crud/query>
+   Update Documents </crud/update>
+   Replace Documents </crud/replace>
+   Delete Documents </crud/delete>
+   Bulk Write Operations </crud/bulk-write>
+   Transactions </crud/transaction>
+   Configure CRUD Operations </crud/read-write-pref>
+   Store Large Files </crud/gridfs>
+   Operations with Builders </crud/builders>
+
+CRUD (Create, Read, Update, Delete) operations enable you to work with
+data stored in MongoDB.
```

Comment:
S: add a sentence introducing the list
- "Visit the following guides to learn more about performing CRUD operations on the data in your MongoDB collections... etc"